src/systems/filecoin_blockchain/struct/block/_index.md

```diff
@@ -7,5 +7,66 @@ The Block is a unit of the Filecoin blockchain.
 
 {{< readfile file="block.id" code="true" lang="go" >}}
 
-{{< readfile file="block.go" code="true" lang="go" >}}
+{{< readfile file="election.id" code="true" lang="go" >}}
+
+# Block syntax validation
+
+Syntax validation refers to validation that may be performed on a block and its messages 
+without reference to outside information such as the parent state tree.
+
+An invalid block must not be transmitted or referenced as a parent.
+
+A syntactically valid block header must decode into fields matching the type definition below. 
+
+A syntactically valid header must have:
+
+- a non-empty array of `Parents` CIDs if `Epoch` is greater than zero,
+- a non-negative `ParentWeight`,
+- a `Miner` address which is an ID-address,
+- a non-negative `Epoch`,
+- a positive `Timestamp`,
+- a `Ticket` with non-empty `VRFResult`,
+- an `ElectionProof` with non-empty `VRFResult`.
+
+A syntactically valid full block must have:
+
+- all referenced messages syntactically valid,
+- all referenced parent receipts syntactically valid,
+
+Note that validation of the block signature requires access to the miner worker address from the
+parent tipset state, so signature validation forms part of sematic validation. Similarly, message 
+signature validation requires lookup of the public key associated with each messages `From`
+account actor in the block's parent state.
+
+# Block semantic validation
+
+Semantic validation refers to validation that requires reference to information outside the block
+header and messages themselves, in particular the parent tipset and state on which the block is built.
+
+A semantically valid block must have:
+
+- `Parents` listed in lexicographic order of their header's `Ticket`,
+- `Parents` all reference valid blocks and form a valid tipset,
+- `ParentStateRoot` identifying the state tree produced by executing the parent tipset's messages,
+- `ParentReceiptRoot` identifying the receipt list produced by parent tipset execution, with one receipt for each non-duplicate message from the parent tipset, 
+- `ParentWeight` matching the weight of the chain up to the parent tipset,
+- `Epoch` greater than that of its parents,
+- `Miner` that is active in the storage power table in the parent tipset state,  
+- a `Ticket` derived from the minimum ticket from the parent tipset's block headers, 
+    - `Ticket.VRFResult` validly signed by the `Miner` actor's worker account public key,
+- `ElectionProof` that is a winner given the `Miner` actor's storage power fraction in the parent state, 
+    - `ElectionProof.VRFResult` validly signed by the `Miner` actor's worker account public key,
+- a `Timestamp` in seconds lying within the quantized window implied by the genesis block's timestamp and the
+block's `Epoch`, and not in the future,
```

Comment:
May want an extra one here per @ZenGround0 's ongoing work.

Reply:
nit: maybe move part of this to the epoch, st the verif is that it is larger than its parents and corresponds to the timestamp (then have extra check in timestamp re clock synchronization).

That would mirror what we would get were we to use a VDF (ie lighter timestamp verif, and added epoch verif)

Reply:
The timestamp validation fits neatly into two rules
(1) the timestamp fits within the epoch boundary 
(2) the block's epoch is not in the future as determined by the node's local reading of the current epoch using its clock.

Reply:
Done, moved the "future" up to the epoch line.